src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/models/ReceivedZipFileData.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services.reports.models;
+
+import io.vavr.Tuple2;
+
+import java.util.List;
+
+public class ReceivedZipFileData {
+    public final Tuple2 zipFileIdentifier;
+    public final List<String> scannableItemDcns;
+    public final List<String> paymentDcns;
+
+    public ReceivedZipFileData(
+        Tuple2 zipFileIdentifier,
```

Comment:
perhaps we need flattened values here ;)

Reply:
do you mean - single line of code? to be honest I like this format, if the line becomes too long it is difficult to read without scrolling

Reply:
I mean flatten the `Tuple2` :wink: